metrics/prometheus.go

```diff
@@ -537,6 +543,26 @@ const (
 	containerEnvPrefix   = "container_env_"
 )
 
+// DefaultContainerLabels implements ContainerLabelsFunc. It exports the
+// container name, first alias, image name as well as all its env and label
+// values.
+func DefaultContainerLabels(container *info.ContainerInfo) map[string]string {
+	set := map[string]string{"id": container.Name}
```

Comment:
I extended the documentation a bit, but I'm not completely sure what you'd like to see here. I had hoped the comments on ContainerLabelsFunc, NewPrometheusCollector and DefaultContainerLabels were helpful. Please let me know what else you'd like to see added here.